addon/components/frost-button.js

```diff
@@ -100,7 +122,15 @@ export default Ember.Component.extend({
    * ['small', 'medium', 'large', 'extra-large']
    * @type {String}
    */
-  size: 'medium',
+  size: '',
```

Comment:
Could potentially have addressed this with internal `_priority` set to default and external `priority` setting that value/being checked against the assertion, but this is already merged...so ;)

In the future (once this is in core) - I want multiple reviewers on any #major# changes